templates/windows-sensu-rabbitmq.json.j2

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Oh woah, so, Window's RabbitMQ uses a different type of config?

Reply:
> Oh woah, so, Window's RabbitMQ uses a different type of config?

It looks like a valid sensu config for rabbitmq connection though I did not pull up another to verify. This is not a config file for actual rabbitmq which I believe it has to either use the old erlang config format (which is miserable) or the new one released that only partially works.